src/tabs/test/tabs.spec.js

```diff
@@ -98,6 +98,10 @@ describe('tabs', function() {
       titles().eq(1).find('> div').click();
       expect(scope.deselectFirst).toHaveBeenCalled();
     });
+
+    it('should expose controller as "tab" scope variable', function() {
+      expect(typeof elm.find('.uib-tab').first().isolateScope().tab).toEqual('object');
```

Comment:
Check how some of the other tests for the other components is testing for controllerAs working - those are more useful guides as far as a readable test for this feature are written.